packages/manager/modules/vps/src/dashboard/rebuild/vps-rebuild.controller.js

```diff
@@ -9,9 +9,21 @@ export default class VpsRebuildController {
   $onInit() {
     this.vpsOptions = {
       doNotSendPassword: false,
+      installRTM: true,
     };
   }
 
+  rtmAvailable() {
+    const selectedImage = this.availableImages.find(
+      (elem) => elem.id === this.vpsOptions.imageId,
+    );
+    return (
+      selectedImage &&
+      !selectedImage.name.toLowerCase().includes('windows') &&
+      !selectedImage.name.toLowerCase().includes('bsd')
+    );
```

Comment:
```suggestion
    [‘windows', 'bsd'].includes(!selectedImage?.name.toLowerCase())
```

Reply:
selectedImage.name n'est pas strictement egal à "windows" ou "bsd".

Reply:
Yes you're right, it's been a long day 😄